downstream/modules/platform/con-aap-migration-prepare.adoc

```diff
@@ -0,0 +1,12 @@
+[id="aap-migration-prepare"]
+
+= Preparing for migration
+
+[role="_abstract"]
+
+Before migrating your current {PlatformNameShort} deployment to {OperatorPlatform}, you need to backup your existing data, create k8s secrets for your secret key and postgresql configuration.
```

Comment:
This is very minor suggestion, but thought it was worth mentioning (verb back up vs. noun backup or back-up):

```suggestion
Before migrating your current {PlatformNameShort} deployment to {OperatorPlatform}, you need to back up your existing data, create k8s secrets for your secret key and postgresql configuration.
```

Reply:
Fixed.